src/_zkapauthorizer/tests/storage_common.py

```diff
@@ -154,7 +154,7 @@ def get_passes(message, num_passes):
 
 def get_passes(message, count, signing_key):
     """
-    :param unicode message: Request-binding message for PrivacyPass.
+    :param bytes message: Request-binding message for PrivacyPass.
```

Comment:
It looks like `make_passes` still believes it is receiving the message as `unicode` (it tries to encode the message using UTF-8).

Reply:
Hmm, not sure why I changed this; reverted.